packages/api/src/routes/interaction.ts

```diff
@@ -149,13 +154,125 @@ const interactions: FastifyPluginAsync = async (fastify): Promise<void> => {
           to: toPlayer.username,
           points,
           timestamp: currentTimestamp,
+          socialsFrom: fromPlayer.socials?.share
```

Comment:
If the player updates their contact info, will other players get the updated info?